students/yixingxu/lesson08/circle.py

```diff
@@ -0,0 +1,39 @@
+from math import pi
```

Comment:
the `__init__.py` file would be on the same level as `circle.py` in the directory